src/lib/Tab/LocationView/LocationsTab.php

```diff
@@ -91,12 +92,14 @@ public function renderView(array $parameters): string
         $formLocationAdd = $this->createLocationAddForm($location);
         $formLocationRemove = $this->createLocationRemoveForm($location, $locations);
         $formLocationSwap = $this->createLocationSwapForm($location);
+        $formLocationSetMain = $this->createLocationSetMainForm($contentInfo, $location);
 
         $viewParameters = [
             'locations' => $locations,
             'form_content_location_add' => $formLocationAdd->createView(),
             'form_content_location_remove' => $formLocationRemove->createView(),
             'form_content_location_swap' => $formLocationSwap->createView(),
+            'form_content_location_set_main' => $formLocationSetMain->createView(),
```

Comment:
If you decided to streamline names to use `update` instead of `set` action name, please also update all occurrences like the ones above: array key and variable name.